### PR TITLE
Enhance code quality and testing

### PR DESCRIPTION
**Pull request by bastelfreak**
- move settings from tox.ini to .prospector.yml
- add a huge bunch of missing docs
- replace invalid variable names
- add missing empty lines

### CODE REVIEW

**Comment by bastelfreak** [comment]
this partly implements https://github.com/virtapi/marmoset/issues/84

File: marmoset/config.py [hunk 1]
```diff
@@ -52,6 +55,7 @@ def read_file(file_path=None):
 
 
 def load(file_path=None):
```
Comment (taxus13) [inline marmoset/config.py]: maybe call the function "load_config" then?

File: marmoset/dhcp/isc_dhcp_ldap_config.py [hunk 2]
```diff
@@ -52,6 +58,7 @@ def save(self):
 
     @staticmethod
     def all():
```
Comment (taxus13) [inline marmoset/dhcp/isc_dhcp_ldap_config.py]: maybe call the function "get_all_db_entries"?

File: marmoset/installimage/__init__.py [hunk 3]
```diff
@@ -1,8 +1,9 @@
-"""initial file for dealing with installimage configs"""
+"""Initial file for dealing with installimage configs"""
 from .installimage_config import InstallimageConfig
 
 
 def create(args):
```
Comment (taxus13) [inline marmoset/installimage/__init__.py:5]: "write_config_file"?

Comment (bastelfreak) [inline marmoset/installimage/__init__.py:5]: comment is wrong, fixed in b1666eb

**Pull request merged**